### ISSUE TITLE
Connect Termination: Cable matching query does not exist

### ISSUE THREAD

**Issue by swoga** [netbox-community/netbox#6636]
### NetBox version

v2.11.7

### Python version

3.8

### Steps to Reproduce

1. Create circuit (https://demo.netbox.dev/circuits/circuits/32/)
2. Connect interface to Termination A
3. Disconnect Termination A
4. Connect same interface to Termination A -> exception

![image](https://user-images.githubusercontent.com/3697291/122755929-feb27380-d295-11eb-8bef-9e6e2d624eeb.png)

### Expected Behavior

Termination A connected to interface.

### Observed Behavior

```
<class 'dcim.models.cables.Cable.DoesNotExist'>

Cable matching query does not exist.

Python version: 3.8.5
NetBox version: 2.11.7
```

**Comment by jeremystretch** [comment]
Yet another caching bug. I'm seriously considering ripping out caching entirely in the next release.

**Comment by swoga** [comment]
Thanks @jeremystretch! This did the trick.

**Comment by AnneWielis** [comment]
Good to know, just came across the same problem. Drastically shortened cachiong time, works like a charm.

**Comment by jeremystretch** [comment]
Marking this as blocked by #6639.

**Comment by jeremystretch** [comment]
This will be fixed in v3.0 with the removal of queryset caching. The workaround in the interim is to set `CACHE_TIMEOUT = 0` in `configuration.py`.
